backend/src/typeorm/repository/User.ts

```diff
@@ -43,4 +44,47 @@ export class UserRepository extends Repository<User> {
       )
       .getMany()
   }
+
+  async findBySearchCriteriaPaged(
+    searchCriteria: string,
+    currentPage: number,
+    pageSize: number,
+  ): Promise<[User[], number]> {
+    return await this.createQueryBuilder('user')
+      .where(
+        'user.firstName like :name or user.lastName like :lastName or user.email like :email',
+        {
+          name: `%${searchCriteria}%`,
+          lastName: `%${searchCriteria}%`,
+          email: `%${searchCriteria}%`,
+        },
+      )
+      .take(pageSize)
+      .skip((currentPage - 1) * pageSize)
+      .getManyAndCount()
+  }
+
+  async findBySearchCriteriaPagedNotActivated(
+    searchCriteria: string,
+    currentPage: number,
+    pageSize: number,
+  ): Promise<[User[], number]> {
+    return await this.createQueryBuilder('user')
+      .where(
+        new Brackets((qb) => {
+          qb.where(
+            'user.firstName like :name or user.lastName like :lastName or user.email like :email',
+            {
+              name: `%${searchCriteria}%`,
+              lastName: `%${searchCriteria}%`,
+              email: `%${searchCriteria}%`,
+            },
+          )
+        }),
+      )
+      .andWhere({ emailChecked: false })
+      .take(pageSize)
+      .skip((currentPage - 1) * pageSize)
+      .getManyAndCount()
```

Comment:
It might make sense to define those queries outside(`backend/src/graphql/resolver/AdminResolver.ts`) and add dynamically to the query builder in oder to simplify things and allow generic filtering

```
const queryBuilder = entity.createQueryBuilder()...

if(notActivated){
   queryBuilder.andWhere({ emailChecked: false })
}

if(someOtherCriteria){
   queryBuilder.andWhere(someOtherCriteria)
}
```

This way we would have a dynamic filtering and adding multiple filter conditionas at least with and would be included right away.

It would also reduce code

Reply:
Die beiden lint Fehler sind übrig nachdem ich yarn lint --fix haben laufen lassen. 
Und wenn ich das tue was da steht gibt es wieder eine neue Fehlermeldung.
Lint scheint mit diesem Code nicht klar zu kommen: 
```ts
[users, count] = await userRepository.findBySearchCriteriaPagedNotActivated(
        searchText,
        currentPage,
        pageSize,
      )
```
Es empfiehlt:
```ts
;[users, count] = await userRepository.findBySearchCriteriaPagedNotActivated(
        searchText,
        currentPage,
        pageSize,
      )
```
Was auch kein Sinn macht.